server/src/main/java/com/datastrato/gravitino/server/web/filter/AdminFilter.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.server.web.filter;
+
+import com.datastrato.gravitino.Entity;
+import com.datastrato.gravitino.authorization.AuthorizationUtils;
+import com.datastrato.gravitino.authorization.Privilege;
+import com.datastrato.gravitino.authorization.Privileges;
+import com.datastrato.gravitino.authorization.SecurableObject;
+import com.datastrato.gravitino.authorization.SecurableObjects;
+import com.datastrato.gravitino.exceptions.ForbiddenException;
+import com.datastrato.gravitino.server.authorization.NameBindings;
+import com.datastrato.gravitino.server.web.Utils;
+import com.datastrato.gravitino.server.web.rest.OperationType;
+import com.google.common.collect.Lists;
+import java.io.IOException;
+import java.util.List;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+@NameBindings.AdminInterface
+public class AdminFilter implements BasedRoleFilter {
```

Comment:
Is it service admin or metalakeadmin, I think you should clearly define it.

Reply:
I added.